pkg/controller/wellknownlocations/wellknownlocations.go

```diff
@@ -21,6 +21,9 @@ const (
 	// IgnoreWgetPowerShellPath contains the path of the powershell script which allows wget to ignore certs. The
 	// container image should already have this mounted
 	IgnoreWgetPowerShellPath = payloadDirectory + "/powershell/wget-ignore-cert.ps1"
+	// HNSPSModule is the path to the powershell module which defines various functions for dealing with Windows HNS
+	// networks
+	HNSPSModule = payloadDirectory + "/powershell/hns.psm1"
```

Comment:
would need to add this to the list of required files and check in checkIfRequiredFilesExist()